force-app/main/default/lwc/reviewSessions/reviewSessions.js

```diff
@@ -201,14 +212,15 @@ export default class ReviewSessions extends LightningElement {
             },
             {
                 label: "",
+                cellAttributes: { class: { fieldName: "alreadyCreatedCSSClass" } },
```

Comment:
My only hesitation on using the background color is that we do not do the same thing with the existing program engagements.